frontend/src/__tests__/cypress/cypress/pages/aboutDialog.ts

```diff
@@ -19,6 +19,10 @@ export class AboutDialog {
     return cy.findByTestId('about-text');
   }
 
+  findImageByAltText(altText: string): Chainable<JQuery<HTMLElement>> {
+    return cy.get(`img[alt*="${altText}"]`);
+  }
```

Comment:
should use `cy.findByAltText`.

Reply:
Why not by Image too ?
Shall I add it to a more common lib ?

Reply:
I suppose if you want to specifically ensure that this is an `img` tag then this can stay. I don't know of another way to assert the element is an image.

BTW using `cy.get` has the entire page as the context. We should be searching just in the modal itself or a sub section of the modal.

Reply:
Yes, it's specificly for the image. For RHOAI and ODH the image alt is according to product name.